pkg/webhook/admission/pod/storage_initializer_injector_test.go

```diff
@@ -202,6 +202,192 @@ func TestStorageInitializerInjector(t *testing.T) {
 				},
 			},
 		},
+
+		"StorageInitializerInjectedReadOnlyUnset": {
```

Comment:
can add tests under the [directVolumeMountForPvc](https://github.com/kserve/kserve/blob/b3019774251f8c64d3c1863c6564c2ef7b6592b5/pkg/webhook/admission/pod/storage_initializer_injector_test.go#L2085) scenario as well?